packages/agent/src/requests.ts

```diff
@@ -11,7 +11,7 @@ export function enable (): boolean {
 export function get (
   url: string,
   query: string[],
-  onerror: ErrorEventHandler
+  onerror: OnErrorEventHandler
```

Comment:
changed type name in lib.dom.d.ts

https://github.com/Microsoft/TypeScript/commit/00bf32ca3967b07e8663d0cd2b3e2bbf572da88b#diff-46fd87925e4552c166ec188712741c3fL5478